src/Model/DeleteManager.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Model;
+
+use PDO;
+use PDOException;
+
+class DeleteManager extends AbstractManager
```

Comment:
The idea of managers is to have one Manager class per database table.
So this probably should be a BookManager.
And if you don't destroy the AbstractManager and have your BookManager extends the AbstractManager (as supposed by the framework), you don't even have to add a delete method because you'll inherit the delete from AbstractManager